.eslintignore

```diff
@@ -0,0 +1,3 @@
+build
+node_modules
+*.spec.js
```

Comment:
`test` files are code as well, no reason to make them inconsistent and less readable. exclusion rules can be created if there's something especially painful to deal with within `spec` files

Reply:
Yes good point @rolandas-valantinas I shall change it.